lib/src/room.dart

```diff
@@ -159,10 +160,16 @@ class KurentoRoom extends ChangeNotifier implements Room {
   }
 
   // The audio is muted if the first audio track is disabled.
-  bool get _isAudioMuted => _localStream!.getAudioTracks().isEmpty ? false : !_localStream!.getAudioTracks()[0].enabled;
+  bool get _isAudioMuted => _localStream!.getAudioTracks().isEmpty ? true : !_localStream!.getAudioTracks()[0].enabled;
```

Comment:
I have a fix on dash which I'll commit which makes this experience much better. This let the agent know that the Explorer is not going to share the video in a nice way.